app/backend/prisma/migrations/20220902151808_workhours_unique_day/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[date,userId,taskId]` on the table `WorkHour` will be added. If there are existing duplicate values, this will fail.
```

Comment:
Let's see if this will work :)

Reply:
I did the work now to transform the data to fit into the unique constraint now, please review. Just did manual tests locally and it works on my machine